### DISCUSSION TITLE
(debunked) trojan warning

### DISCUSSION THREAD

**Discussion by AlexBreault** [IceHacks/SurvivCheatInjector#908] [General]
I used **VirusTotal** to scan the zip file, and it contains a **JS Trojan** that can **infect web browsers** using SQL and **steal personal information.** It also has the ability to **compromise system files.** It is highly and **rarely detected.** If you have recently downloaded this file, remove it and do a **full scan** of your computer. ### **You’ve been warned.**

Search this: **0bedcf393792019da5a16034c95b3796a1f785cc689bbef3520d66a86f6505d6** on VirusTotal.

Or Look Here: https://www.virustotal.com/gui/file/0bedcf393792019da5a16034c95b3796a1f785cc689bbef3520d66a86f6505d6/detection

**Comment by humphreygaming** [comment]
> JS.Trojan.Iframe.t: When the user opens the page in a web browser, the code is automatically run.

You know that's the _point_ of the cheat, right? It's supposed to run when you open surviv.io in order to "inject" the code into the site, hence the name Surviv.io Cheat **Injector**.  I also wouldn't pay much attention if only 1/55 of the scans flagged it as the very thing it's supposed to be doing anyway.

**Comment by IceHacks** [comment]
"using SQL" lol. This dude either has 20IQ or is just pulling out random jargon for malicious reasons.

**Comment by IceHacks** [comment]
![image](https://user-images.githubusercontent.com/38994479/163686721-bd9f85af-58a5-496d-bfcb-8da4f464d941.png)

How to spot a fake: my cheat is not 5MB big.

**Comment by IceHacks** [comment]
Here is an actual cheat VirusTotal if one is interested.

https://www.virustotal.com/gui/file/0aa18eccb9245ab43d16fa880ecf910ae5695746a167e343d79ec41af9363935/details
